### PR TITLE
Added metrics for chaindata fetcher

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- Added several metrics for chaindata fetcher.
- Status API is implemented to check the fetching is running or not.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: datasync/chaindatafetcher/metrics.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	tokenTransferInsertionTimeGauge = metrics.NewRegisteredGauge("chaindatafetcher/insertion/tokentransfer/gauge", nil)
+	contractsInsertionTimeGauge     = metrics.NewRegisteredGauge("chaindatafetcher/insertion/contracts/gauge", nil)
+	tracesInsertionTimeGauge        = metrics.NewRegisteredGauge("chaindatafetcher/insertion/traces/gauge", nil)
+	totalInsertionTimeGauge         = metrics.NewRegisteredGauge("chaindatafetcher/insertion/total/gauge", nil)
+
+	handledBlockNumberGauge = metrics.NewRegisteredGauge("chaindatafetcher/handle/blocknumber/gauge", nil)
+
+	numChainEventGauge = metrics.NewRegisteredGauge("chaindatafetcher/chainevent/gauge", nil)
+	numRequestsGauge   = metrics.NewRegisteredGauge("chaindatafetcher/requests/gauge", nil)
+)
```
Comment (KimKyungup) [inline datasync/chaindatafetcher/metrics.go:18]: How about add retryCounter? :)

File: datasync/chaindatafetcher/metrics.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import "github.com/rcrowley/go-metrics"
+
+var (
+	checkpointGauge = metrics.NewRegisteredGauge("chaindatafetcher/checkpoint/gauge", nil)
+
+	txInsertionTimeGauge            = metrics.NewRegisteredGauge("chaindatafetcher/insertion/tx/gauge", nil)
+	tokenTransferInsertionTimeGauge = metrics.NewRegisteredGauge("chaindatafetcher/insertion/tokentransfer/gauge", nil)
+	contractsInsertionTimeGauge     = metrics.NewRegisteredGauge("chaindatafetcher/insertion/contracts/gauge", nil)
+	tracesInsertionTimeGauge        = metrics.NewRegisteredGauge("chaindatafetcher/insertion/traces/gauge", nil)
```
Comment (aidan-kwon) [inline datasync/chaindatafetcher/metrics.go:11]: Why some are singular and others are plural?

File: datasync/chaindatafetcher/metrics.go (hunk 1 above)
Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/metrics.go:23]: Applied. 59464c83bacc45f049251e25bc280f53eb731c4d

File: datasync/chaindatafetcher/metrics.go (hunk 2 above)
Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/metrics.go]: Applied. fa42787378437f56b89c9f5203bbf060bee5f285

**Review by aidan-kwon — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]

**Comment by jeongkyun-oh** [comment]
@ehnuje PTAL

**Review by ehnuje — APPROVED** [review]

**Pull request merged**